src/components/advanced-color-picker/advanced-color-picker.d.ts

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+
+export interface AdvancedColorPickerPropTypes {
+  open?: boolean;
+  availableColors?: [];
+  selectedColor?: string;
+  defaultColor?: string;
+  onChange?: (ev: React.ChangeEvent<HTMLElement>) => void;
+  onOpen?: (ev: React.ChangeEvent<HTMLElement>) => void;
```

Comment:
It's not always React Synthetic event -> check line 129 of `simple-color-picker.component.js`

Reply:
indeed, well spotted, thanks! fixed.